app/models/project.rb

```diff
@@ -19,7 +19,8 @@ class Project < ActiveRecord::Base
 
   has_many :comments, as: :commentable, dependent: :destroy
 
-  has_and_belongs_to_many :episodes
+  has_many :episode_project_associations
+  has_many :episodes, through: :episode_project_associations
```

Comment:
I need the intermediate object — to store `created_at` and sort by it.
This allows me to add existing project on top of the RSS feed, if it was just added in the episode.